src/CMakeLists.txt

```diff
@@ -182,13 +105,8 @@ endforeach()
 
 target_compile_definitions(latte PRIVATE XSYEV ${PRECISION}PREC GPU${GPUOPT} DBCSR_${DBCSR_OPT} MPI_${DO_MPI} PROGRESS${PROGRESS})
 
-if(DO_MPI)
-  set(LATTE LATTE_MPI_${PRECISION})
-else()
-  set(LATTE LATTE_${PRECISION})
-endif()
 add_executable(${LATTE} ${CMAKE_CURRENT_SOURCE_DIR}/../src/latte.F90)
 target_link_libraries(${LATTE} latte ${PROGRESS_LIBRARIES})
-target_include_directories(${LATTE} PRIVATE ${MPI_Fortran_INCLUDE_PATH} ${PROGRESS_INCLUDE_DIRS} ${BML_INCLUDE_DIRS})
-target_compile_definitions(${LATTE} PRIVATE XSYEV ${PRECISION}PREC GPU${GPUOPT} DBCSR_${DBCSR_OPT} MPI_${DO_MPI} PROGRESS${PROGRESS})
-install(TARGETS ${LATTE} DESTINATION ${CMAKE_INSTALL_BINDIR})
+
+target_include_directories(latte PRIVATE ${MPI_Fortran_INCLUDE_PATH} ${PROGRESS_INCLUDE_DIRS} ${BML_INCLUDE_DIRS})
+target_compile_definitions(latte PRIVATE XSYEV ${PRECISION}PREC GPU${GPUOPT} DBCSR_${DBCSR_OPT} MPI_${DO_MPI} PROGRESS${PROGRESS})
```

Comment:
@jeanlucf22 then something is wrong here. These defines and install rules were for the execuable.

Reply:
Building the executable uses the .a and does not need the includes, I think. Is it failing for you?

Reply:
I haven't checked, but back when at least the defines where using `latte.F90`. We could of course make the defines `PUBLIC` on the library, then all consumers would get them too.

Reply:
I don't know the history of the cmake build in LATTE, but it certainly was not working for me before I made these changes.

Reply:
It worked back in 2020 ;-)